docs/integrations/sources/uppromote.md

```diff
@@ -0,0 +1,26 @@
+# UpPromote
+The Uppromote Connector for Airbyte enables seamless data integration between Uppromote, an affiliate and influencer marketing platform, and your data warehouses or analytics tools.
+
+## Configuration
+
+| Input | Type | Description | Default Value |
+|-------|------|-------------|---------------|
+| `api_key` | `string` | API Key. For developing your own custom integration with UpPromote, you can create an API key. This is available from Professional plan.  Simply go to Settings &gt; Integration &gt; API &gt; Create API Key.  |  |
+
+## Streams
+| Stream Name | Primary Key | Pagination | Supports Full Sync | Supports Incremental |
+|-------------|-------------|------------|---------------------|----------------------|
+| affiliates | id | DefaultPaginator | ✅ |  ❌  |
+| coupons | id | DefaultPaginator | ✅ |  ❌  |
+| referrals | id | DefaultPaginator | ✅ |  ❌  |
```

Comment:
Referrals you can implement incremental https://aff-api.secomapp.com/docs/index.html#get-list-referrals can you check it? Thanks.